lib/modules/settings/screens/notifications/pre_athan/pre_athan_settings_screen.dart

```diff
@@ -0,0 +1,220 @@
+import 'dart:async';
```

Comment:
Maybe for future, but I think we can merge the 3 types of `notification screen` into one base, and only implement the differences, a lot of copy paste in these files

Reply:
Done